<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementWindow</class>
 <widget class="QWidget" name="MeasurementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Record Measurement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_recordButton">
       <property name="text">
        <string>Record</string>
       </property>
       <property name="icon">
        <iconset theme="media-record"/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_recordPath">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>D:/</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_statusMocap">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { 
color : red; 
background-color : rgb(255, 200, 200);
}</string>
       </property>
       <property name="text">
        <string>Not Connected</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Mocap Status</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>A-mode Status</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_recordPath">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="folder-open"/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="label_statusAmode">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { 
color : red; 
background-color : rgb(255, 200, 200);
}</string>
       </property>
       <property name="text">
        <string>Not Connected</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Intermediate Record Status</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="label_statusIntermRec">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { color : rgb(255, 85, 0); background-color : rgb(255, 255, 127); }</string>
       </property>
       <property name="text">
        <string>Idle</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
